server/src/main/java/org/opensearch/action/admin/cluster/node/stats/NodeStats.java

```diff
@@ -152,6 +156,12 @@ public NodeStats(StreamInput in) throws IOException {
         } else {
             searchBackpressureStats = null;
         }
+
+        if (in.getVersion().onOrAfter(Version.V_2_6_0)) {
+            clusterManagerThrottlingStats = in.readOptionalWriteable(ClusterManagerThrottlingStats::new);
+        } else {
+            clusterManagerThrottlingStats = null;
+        }
```

Comment:
Implicitly it should be null?

Reply:
Yes, it should be null as those stats will not be present in other version so.

Reply:
Yes, it should be null as those stats will not be present in other version so.